src/api/app/models/bs_request.rb

```diff
@@ -22,12 +22,18 @@ class SaveError < APIException
     setup 'request_save_error'
   end
 
-  scope :to_accept, -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
+  scope :to_accept,
+   -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.